python/mxnet/test_utils.py

```diff
@@ -102,6 +102,16 @@ def random_arrays(*shapes):
     return arrays
 
 
+def random_uniform_arrays(*shapes, low=0.0, high=1.0):
+    """Generate some random numpy arrays."""
+    arrays = [np.array(np.random.uniform(low, high), dtype=default_dtype())
+              if len(s) == 0 else np.random.uniform(low, high, size=s).astype(default_dtype())
+              for s in shapes]
```

Comment:
The if-condition is not necessary. We can directly write
```python
arrays = [np.random.uniform(low, high, size=s) for s in shapes]
```

Reply:
Also, we'd better add a dtype flag to the function, which is None by default and set it to default_dtype() if it's None.

Reply:
I don't like to return different types based on inputs, the function should just return a list of arrays. other than that PR looks fine and approach is good. thanks for the PR.

Reply:
@sxjscience Will do that.

@larroy Agreed even I don't like that. However note that `random_arrays` in `test_utils.py` has that. I wanted `random_uniform_arrays` to be a drop-in replacement for `random_arrays` if and where necessary. But I guess it makes sense to not worry about it.
https://github.com/apache/incubator-mxnet/blob/04c3eec3e8ea793614244e18a89c6b3a13f12d6a/python/mxnet/test_utils.py#L95-L102